.github/workflows/gos-ci.yml

```diff
@@ -59,21 +59,22 @@ jobs:
       release: ${{ steps.release.outputs.release }}
       debian_release_name: ${{ steps.release.outputs.debian_release_name }}
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - name: Gather information
         id: release
         run: |
           RELEASE=$(head -n 1 debian/changelog | cut -d ' ' -f3 | cut -d ';' -f1)
-          echo "::set-output name=release::$RELEASE"
-
+          echo "release=$RELEASE" >> $GITHUB_OUTPUT
```

Comment:
Should we use `>` instead of `>>`, because this seems to be the first write to the variable `GITHUB_OUTPUT`?

Reply:
Possible, but I would keep it as it is.